core/src/main/resources/application-dev.properties

```diff
@@ -1,8 +1,8 @@
 ################################################################################
 # Configuration for development environment
 # Please set below properties!
-spring.datasource.url=jdbc:mysql://${host}/${schema_name}
+spring.datasource.url=jdbc:mysql://referral-1.cdaj903rkno2.us-west-1.rds.amazonaws.com/referral
 spring.datasource.driverClassName=com.mysql.cj.jdbc.Driver
-spring.datasource.username=${user_name}
-spring.datasource.password=${password}
+spring.datasource.username=referral
+spring.datasource.password=Referral&2020
```

Comment:
Actually can you undo this and add this file to the .gitignore file? This credential is not meant to be pushed to github!